content/docs/plugins/notifications-plugin/_index.md

```diff
@@ -1,70 +1,80 @@
+
 ---
 title: "Notifications Plugin"
+id: index
+weight: 1
+description: How to get started with the notifications and signals
+---
 ---
 
-Documentation section of notifications plugin. This plugin is part of Backstage Core.
+The Backstage Notifications System provides a way for plugins and external services to send notifications to Backstage users.
+
+These notifications are displayed in the dedicated page of the Backstage frontend UI or by frontend plugins per specific scenarios.
+
+Additionally, notifications can be sent to external channels (like email) via "processors" implemented within plugins.
 
 Upstream documentation can be found in:
 
-- [Backend plugin](https://backstage.io/docs/reference/plugin-notifications-backend/)
+- [Notifications and Signals](https://backstage.io/docs/notifications/)
 - [Backend email plugin](https://backstage.io/docs/reference/plugin-notifications-backend-module-email/)
 
 # Frontend
-The frontend UI is implemented as a side bar item.  
-The following screenshots show how to use the notifications UI.  
-Whenever a new notification is detected the frontend pops an alert to the user.  
+Notifications are messages sent to either individual users or groups. They are not intended for inter-process communication of any kind. 
+
+To list and manage, choose `Notifications` from the left-side menu item.
 
-## The Frontend UI
+There are two basic types of notifications:
+- Broadcast: Messages sent to all users of Backstage.
+- Entity: Messages delivered to specific listed entities from the Catalog, such as Users or Groups.
 
-### Notifications page
-![Frontend UI](./sidebar-button-plus-alert.png)
 
-### Notifications list
-![Notifications list](./list-view.png)
+![Frontend UI](./notificationsPage.png)
 
 # Backend
 The backend plugin provides the backend application for reading and writing notifications.
 
 ## Authentication
+The Notifications are primarily meant to be sent by backend plugins. In such flow, the authentication is shared among them.
 
-Two kinds of authentation are in place with Notifications Plugin:
-
-1) For reading notifications the JWT token generated by backstage IDP with typ `vnd.backstage.user` is used. The JWT claim `sub` will be the one used to filter the notifications.
+In case an external system (like a Workflow) create a new notification by sending POST request the Notification REST API, authentication needs to be properly configured via setting the `backend.auth.externalAccess` property of the `app-config` .
```

Comment:
Typos, suggest instead: In **a** case **that** an external system (like a Workflow) create**d** a new notification by sending **a** POST request **to** the Notification REST API,